ArduCopter/AP_Arming.cpp

```diff
@@ -934,3 +943,18 @@ bool AP_Arming_Copter::disarm(const AP_Arming::Method method, bool do_disarm_che
 
     return true;
 }
+// check if vertical velocity innovations are less than 1m/s (0.5m/s specifically)
```

Comment:
```suggestion
// check if vertical velocity innovations are less than 0.5m/s
```

 (and adjust elsewhere)